pkgs/development/python-modules/open-garage/default.nix

```diff
@@ -8,15 +8,16 @@
 
 buildPythonPackage rec {
   pname = "open-garage";
-  version = "0.1.6";
+  version = "0.2.0";
+  format = "setuptools";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.6";
```

Comment:
It seems that it technically still supports Python 3.5: https://github.com/Danielhiversen/pyOpenGarage/blob/master/setup.py

But well, don't think it matters too much.

Reply:
It would. Python 3.5 is EOL for a little over a year. I doubt that a lot people are running Home Assistant with Python < 3.8 but you never know.

Reply:
Well, I saw other PR from you that is still set to Python 3.5 and you didn't change it. I would prefer this to be consistent.

Reply:
> It would. Python 3.5 is EOL for a little over a year. I doubt that a lot people are running Home Assistant with Python < 3.8 but you never know.

BTW, this is quite unlikely. We don't package Python 3.5 anymore (https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/default.nix).